be/src/common/config.h

```diff
@@ -287,6 +287,9 @@ CONF_mInt32(repair_compaction_interval_seconds, "600"); // 10 min
 // if compaction of a tablet failed, this tablet should not be chosen to
 // compaction until this interval passes.
 CONF_mInt64(min_compaction_failure_interval_sec, "120"); // 2 min
+
+CONF_mInt64(min_cmumulative_compaction_failure_interval_sec, "30"); // 30s
```

Comment:
why add one new config `min_cmumulative_compaction_failure_interval_sec` instead of `min_compaction_failure_interval_sec` ?

Reply:
The positioning of the cumulative compaction is the `minor` compaction. It should not use the same interval as the base compaction. For example, if the user write at a speed of 10 tps, if the interval is 120s, it will directly cause 1200 versions to fail to be compacted.